test/smoke/multi-image3/Makefile

```diff
@@ -5,14 +5,13 @@ TESTSRC_MAIN = multi-image.c
 TESTSRC_AUX  =
 TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
 
-OMP_FLAGS    = -fopenmp --offload-arch=$(AOMP_GPU)$(AOMP_TARGET_FEATURES) --offload-arch=gfx803$(AOMP_TARGET_FEATURES) --offload-arch=gfx1031$(AOMP_TARGET_FEATURES)
+RUNCMD	     = ./$(TESTNAME) && strings $(TESTNAME) | grep $(AOMP_GPU)$(AOMP_TARGET_FEATURES) && strings $(TESTNAME) | grep gfx908$(AOMP_TARGET_FEATURES) && strings $(TESTNAME) | grep gfx942$(AOMP_TARGET_FEATURES)
+
+OMP_FLAGS    = -fopenmp --offload-arch=$(AOMP_GPU)$(AOMP_TARGET_FEATURES),gfx908$(AOMP_TARGET_FEATURES),gfx942$(AOMP_TARGET_FEATURES)
```

Comment:
Question: Is this set somewhere outside (or is this a usecase that should be supported)?
If not, then I think it would be better to remove it, to not lead to confusion.

Reply:
I'm not quite sure but as it was set in the original test (before my commits modifying it adding xnack as explicit target feature)